client/core/servercontroller.cpp

```diff
@@ -691,7 +691,7 @@ ErrorCode ServerController::isUserInSudo(const ServerCredentials &credentials, D
     const QString scriptData = amnezia::scriptData(SharedScriptType::check_user_in_sudo);
     ErrorCode error = runScript(credentials, replaceVars(scriptData, genVarsForScript(credentials)), cbReadStdOut, cbReadStdErr);
 
-    if (!stdOut.contains("sudo")) return ErrorCode::ServerUserNotInSudo;
+    if (!stdOut.contains("(ALL) NOPASSWD: ALL")) return ErrorCode::ServerUserNotInSudo;
```

Comment:
Using this condition may be not safe
I see output is "(ALL : ALL) NOPASSWD: ALL" on debian

Instead of that we going to send user password to sudo if it asking for it, we will implement it later

Reply:
So, any such fix requires deep testing